airbyte-integrations/connectors/source-metabase/source_metabase/schemas/snippets.json

```
@@ -0,0 +1,67 @@
+{
+  "type": ["null", "object"],
+  "properties": {
+    "description": {
+      "type": ["null", "string"]
+    },
+    "archived": {
+      "type": ["null", "boolean"]
+    },
+    "creator": {
+      "type": ["null", "object"],
+      "properties": {
+        "email": {
+          "type": ["null", "string"]
+        },
+        "first_name": {
+          "type": ["null", "string"]
+        },
+        "last_login": {
+          "type": ["null", "string"]
+        },
+        "is_qbnewb": {
+          "type": ["null", "boolean"]
+        },
+        "is_superuser": {
+          "type": ["null", "boolean"]
+        },
+        "id": {
+          "type": ["null", "integer"]
+        },
+        "last_name": {
+          "type": ["null", "string"]
+        },
+        "date_joined": {
+          "type": ["null", "string"]
+        },
+        "common_name": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "content": {
+      "type": ["null", "string"]
+    },
+    "collection_id": {
+      "type": ["null", "integer"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "creator_id": {
+      "type": ["null", "integer"]
+    },
+    "updated_at": {
+      "type": ["null", "string"]
+    },
+    "id": {
+      "type": ["null", "integer"]
```

```suggestion
      "type": "integer"
```
It's the PK isn't it?